diff --git a/dist/index.js b/dist/index.js
index 6025229a1604995d2a6d14552b65703f55dfa566..d543b7acf0e6b1330d17ef45fcfab4d31f91f170 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -207,10 +207,10 @@ __export(api_exports, {
 var jsonrpc_exports = {};
 
 // src/types/api/index.ts
-var RPCSPEC07 = __toESM(require("@starknet-io/starknet-types-07"));
-var RPCSPEC08 = __toESM(require("@starknet-io/starknet-types-08"));
-var import_starknet_types_08 = require("@starknet-io/starknet-types-08");
-__reExport(api_exports, require("@starknet-io/starknet-types-08"));
+var RPCSPEC07 = __toESM(require("@starknet-io/types-js"));
+var RPCSPEC08 = __toESM(require("starknet-types-08"));
+var import_starknet_types_08 = require("starknet-types-08");
+__reExport(api_exports, require("starknet-types-08"));
 
 // src/utils/encode.ts
 var encode_exports = {};
@@ -637,7 +637,7 @@ var BlockTag = {
 };
 
 // src/provider/types/spec.type.ts
-var RPCSPEC082 = __toESM(require("@starknet-io/starknet-types-08"));
+var RPCSPEC082 = __toESM(require("starknet-types-08"));
 var { ETransactionVersion: ETransactionVersion2 } = RPCSPEC082;
 var { ETransactionVersion2: ETransactionVersion22 } = RPCSPEC082;
 var { ETransactionVersion3 } = RPCSPEC082;
@@ -729,7 +729,7 @@ var OutsideExecutionVersion = {
 };
 
 // src/types/typedData.ts
-var import_starknet_types_07 = require("@starknet-io/starknet-types-07");
+var import_starknet_types_07 = require("@starknet-io/types-js");
 
 // src/utils/json.ts
 var json_exports = {};
diff --git a/dist/index.mjs b/dist/index.mjs
index e0a9a2516c2d6ee3d06b75c61dd203b188ce0a52..15195903b3d64ed6038dd8584a7b2973985655c0 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -63,10 +63,10 @@ var jsonrpc_exports = {};
 
 // src/types/api/index.ts
 __reExport(api_exports, starknet_types_08_star);
-import * as RPCSPEC07 from "@starknet-io/starknet-types-07";
-import * as RPCSPEC08 from "@starknet-io/starknet-types-08";
-import { PAYMASTER_API } from "@starknet-io/starknet-types-08";
-import * as starknet_types_08_star from "@starknet-io/starknet-types-08";
+import * as RPCSPEC07 from "@starknet-io/types-js";
+import * as RPCSPEC08 from "starknet-types-08";
+import { PAYMASTER_API } from "starknet-types-08";
+import * as starknet_types_08_star from "starknet-types-08";
 
 // src/utils/encode.ts
 var encode_exports = {};
@@ -493,7 +493,7 @@ var BlockTag = {
 };
 
 // src/provider/types/spec.type.ts
-import * as RPCSPEC082 from "@starknet-io/starknet-types-08";
+import * as RPCSPEC082 from "starknet-types-08";
 var { ETransactionVersion: ETransactionVersion2 } = RPCSPEC082;
 var { ETransactionVersion2: ETransactionVersion22 } = RPCSPEC082;
 var { ETransactionVersion3 } = RPCSPEC082;
@@ -587,7 +587,7 @@ var OutsideExecutionVersion = {
 // src/types/typedData.ts
 import {
   TypedDataRevision
-} from "@starknet-io/starknet-types-07";
+} from "@starknet-io/types-js";
 
 // src/utils/json.ts
 var json_exports = {};
