diff --git a/dist/index.js b/dist/index.js
index 6025229a1604995d2a6d14552b65703f55dfa566..4a4eb69c96884836e7b422595811581df5ed1753 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -207,7 +207,7 @@ __export(api_exports, {
 var jsonrpc_exports = {};
 
 // src/types/api/index.ts
-var RPCSPEC07 = __toESM(require("@starknet-io/starknet-types-07"));
+var RPCSPEC07 = __toESM(require("@starknet-io/types-js"));
 var RPCSPEC08 = __toESM(require("@starknet-io/starknet-types-08"));
 var import_starknet_types_08 = require("@starknet-io/starknet-types-08");
 __reExport(api_exports, require("@starknet-io/starknet-types-08"));
@@ -729,7 +729,7 @@ var OutsideExecutionVersion = {
 };
 
 // src/types/typedData.ts
-var import_starknet_types_07 = require("@starknet-io/starknet-types-07");
+var import_starknet_types_07 = require("@starknet-io/types-js");
 
 // src/utils/json.ts
 var json_exports = {};
diff --git a/dist/index.mjs b/dist/index.mjs
index e0a9a2516c2d6ee3d06b75c61dd203b188ce0a52..4e0255d297cc533b841c4696cdc388f2a921b973 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -63,7 +63,7 @@ var jsonrpc_exports = {};
 
 // src/types/api/index.ts
 __reExport(api_exports, starknet_types_08_star);
-import * as RPCSPEC07 from "@starknet-io/starknet-types-07";
+import * as RPCSPEC07 from "@starknet-io/types-js";
 import * as RPCSPEC08 from "@starknet-io/starknet-types-08";
 import { PAYMASTER_API } from "@starknet-io/starknet-types-08";
 import * as starknet_types_08_star from "@starknet-io/starknet-types-08";
@@ -587,7 +587,7 @@ var OutsideExecutionVersion = {
 // src/types/typedData.ts
 import {
   TypedDataRevision
-} from "@starknet-io/starknet-types-07";
+} from "@starknet-io/types-js";
 
 // src/utils/json.ts
 var json_exports = {};
